### PR TITLE
3296: Added on waiting list state

### PR DESCRIPTION
**Pull request by cableman**
Added new state for place2book tickets - on waiting list.

See https://platform.dandigbib.org/issues/3296

P.S. Also did a little coding style clean-up, but just the worst of it.

### CODE REVIEW

**Comment by cableman** [comment]
The laste Scrutinizer notifications is from old code.

File: modules/ding_place2book/ding_place2book.module [hunk 1]
```diff
@@ -1079,52 +1099,45 @@ function ding_place2book_render_place2book_ticketsinfo($node) {
...
+      $output = '<div class="place2book-tickets-label">'. t('Tickets') . '</div><div class="place2book-tickets-info">' . t('Sold out') . '</div>';
       break;
+
     case 'order-link':
-      $output = '<div class="place2book order">' . l(t('Book a ticket'), $url) . '</div>';
+      $output = l(t('Book a ticket'), $url, array('attributes' => array('class' => array('ticket-available', 'button'))));
+      break;
+
+    case 'sale-not-started':
+      $output = '<div class="place2book-tickets-label">'. t('Tickets') . '</div><div class="place2book-tickets-info">' . t('Ticket sale has not yet started for this event') . '</div>';
```
Comment (kasperg) [inline modules/ding_place2book/ding_place2book.module]: `sale-not-started` sees to be a new ticket type which is also introduced here but is outside the scope of this PR. Is that intentional?

File: modules/ding_place2book/ding_place2book.module [hunk 2]
```diff
@@ -1079,52 +1099,45 @@ function ding_place2book_render_place2book_ticketsinfo($node) {
...
+ * Default theme function for 'ticketsinfo'.
  */
 function theme_place2book_ticketsinfo($variables) {
   $url = $variables['url'];
   $type = $variables['type'];
 
   switch ($type) {
     case 'event-over':
-      $output = '<div class="place2book info">' . t('The event has already taken place') . '</div>';
+      $output = '<div class="place2book-tickets-label">'. t('Tickets') . '</div><div class="place2book-tickets-info">' . t('The event has already taken place') . '</div>';
```
Comment (kasperg) [inline modules/ding_place2book/ding_place2book.module]: :+1:

**Comment by cableman** [comment]
Rebase on newest master branch

**Pull request merged**